go.mod

```diff
@@ -14,9 +14,9 @@ require (
 	github.com/onsi/gomega v1.10.3
 	github.com/openshift/api v0.0.0-20200331152225-585af27e34fd
 	github.com/openshift/client-go v0.0.0-20200326155132-2a6cd50aedd0
-	github.com/operator-framework/api v0.5.1
+	github.com/operator-framework/api v0.5.0
```

Comment:
Same here - this looks like a relatively stale API version.

Reply:
This should be the placeholder `v0.0.0-00010101000000-000000000000` version, just to indicate the submodule status of the dependency

Reply:
I need to wrap my head around whether we care about differing API tags between the registry and OLM components. Is it sufficient to how those overall packages share a single API version, or is it possible down the line that we'll need something like v0.5.x for the registry and v0.6.x for OLM?